core/server/models/newsletter.js

```diff
@@ -38,6 +38,11 @@ const Newsletter = ghostBookshelf.Model.extend({
         return this.hasMany('Post');
     },
 
+    // Force active newsletters for content API
+    enforcedFilters: function enforcedFilters(options) {
+        return options.context && options.context.public ? 'status:active' : null;
```

Comment:
Question: does this evaluate as
`(options.context && options.context.public) ? 'status:active' : null;`
or
`options.context && (options.context.public ? 'status:active' : null);`
?

Reply:
Since`&&` has a higher precedence, think it would be `(options.context && options.context.public) ? 'status:active' : null;` , but I'll update this either ways to remove any ambiguity :)